### ISSUE TITLE
Problems using Fontforge with Monaco.ttf on the Apple Mac

### ISSUE THREAD

**Issue by carlca** [fontforge/fontforge.github.io#123]
macOS 10.13.5
Python 2.17.15
FontForge 09.31 UTC 21-Mar-2018 (presumably this timestamp acts as the version)

Hello, ultimately I am trying to use `FontForge` in conjunction with a tool called `Ligaturizer` which can be found at https://github.com/ToxicFrog/Ligaturizer. I have installed FontForge using `brew install fontforge --with-python` and have confirmed that I can `import fontforge` within Python 2.7.15. However, if I try to reproduce part of what `Ligaturizer` does by typing `fontforge.open("./input-fonts/Monaco.ttf")`, I get the following error...

![screen shot 2018-05-20 at 12 33 11](https://user-images.githubusercontent.com/260612/40278518-0f79a188-5c2a-11e8-8534-be5bb41555bb.png)

Can anyone help? Thanks, Carl'

**Comment by JoesCat** [comment]
Hi @carlca,
What I believe you are running into here is lack of a python development package.

You installed python, but for fontforge to be able to interact with python, you will need to build fontforge with python's development package (These would be python *.h files and other things needed to link to python so that fontforge can "speak python").
The documentation isn't particularly strong in this area, and the build system needs updating to catch this. Right now, there is a patch waiting to be reviewed/tested here:
https://github.com/fontforge/fontforge/pull/3292

I was going to suggest you test it out, but I'm looking at homebrew's [fontforge.rb,](https://github.com/Homebrew/homebrew-core/blob/master/Formula/fontforge.rb) and it does not appear to have anything to build HEAD, just tag point 20170731. If it did, I would suggest copying the two files [configure.ac](https://github.com/JoesCat/fontforge/blob/pythonpkg/configure.ac) and [m4/fontforge_arg_enable.m4](https://github.com/JoesCat/fontforge/blob/pythonpkg/m4/fontforge_arg_enable.m4) from here [python test branch](https://github.com/JoesCat/fontforge/tree/pythonpkg), so that you could try test them out.

Start with adding the development files or package for python first, then rebuild fontforge again.

**Comment by carlca** [comment]
Thanks for the reply. Are you saying that the `brew install fontforge --with-python` wasn't sufficient?
Which files specifically do I need to download and install in order to follow your final suggestion?

**Comment by jtanx** [comment]
No, this has nothing to do with python devel. Don't bother messing around with any of that.

As you are able to import fontforge, and you're getting a traceback, that clearly shows that it's being compiled just fine. 

From the traceback, it's reaching here: https://github.com/fontforge/fontforge/blob/master/fontforge/python.c#L1165

Which suggests there's a problem with your font, and FontForge can't open it. Providing a sample would help track down the exact cause.

**Comment by carlca** [comment]
Thanks, Jeremy. The font in question is the standard Monaco.ttf font which is supplied with macOS. Licencing laws, presumably, forbid me from attaching it here. I have, however, managed to get `fontforge` to run with a couple of Monaco alternatives, which I have attached...

[Monaco_Linux.ttf.zip](https://github.com/fontforge/fontforge.github.io/files/2022224/Monaco_Linux.ttf.zip)

[monaco.zip](https://github.com/fontforge/fontforge.github.io/files/2022222/monaco.zip)

The actual script I'm trying to run with `fontforge` is this one https://github.com/ToxicFrog/Ligaturizer. Full instructions are provided with it.

The problem with the two fonts I attached, is that the script appears to run, but the font's produced do not appear to have any ligatures.

Thanks in advance for your help!
